quickwit/quickwit-doc-mapper/src/default_doc_mapper/mapping_tree.rs

```diff
@@ -1096,4 +1148,23 @@ mod tests {
             serde_json::json!("010203")
         );
     }
+
+    #[test]
+    fn test_field_path_for_field_name() {
```

Comment:
What about string starting or ending with a `.` vs `\.`?

Reply:
Yes... I thought about this. We don't do any validation for tags and timestamp fields, I was going to open an issue on this.